homework-submission/homeworkWeek1/main.jsx

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+const { Component } = React;
+
+// This component renders list items
+const ListItem = props => {
+  const { listData, isStatic, markAsCompleted } = props;
+  let { description, deadline, id, done } = listData;
+  return (
+    <li
+      id={id}
+      onClick={() => {
+        if (isStatic) {
+          const element = document.getElementById(id);
+          element.classList.toggle('done');
+        } else {
+          markAsCompleted(id - 1);
```

Comment:
you have two different functions to define the same job!

For that I would be using another strategy.
Let's say you want to click an element, and hereby we could see the `li` element crossed line.

All we need for that is the element, once we have it we have it's functionality `element.classList.toggle('done');`.
in this Component you have an `id` definition on line 11, using numeric id's, that make it a quite strange for me.
then you are getting it on line 14 with `.getElementById(id)` and it's fine for simplicity sake, but we need simpler.

what about a one function can control both `StaticList` and the `DynaimcList` with no `isStatic` or `markAsCompleted` prop?

it's going to be fun 🙂

**First Step**: We don't need to make a numeric id attribute here!
Please follow the rest of the comments to see what to do next...